pydantic/generics.py

```diff
@@ -0,0 +1,76 @@
+from typing import TYPE_CHECKING, Any, Dict, Tuple, Type, TypeVar, Union, get_type_hints, no_type_check
+
+from pydantic import BaseModel, create_model
+from pydantic.class_validators import gather_validators
+
+
+class GenericModel:
```

Comment:
I figured GenericModel was an appropriate name but I can change it back if desired.